src/main/java/com/cinema/model/dto/MovieRequestDto.java

```diff
@@ -0,0 +1,20 @@
+package com.cinema.model.dto;
+
+public class MovieRequestDto {
+    private String title;
```

Comment:
Maybe we'll need movie description in DTOs as well

Reply:
> Maybe we'll need movie description in DTOs as well

\+